index.html

```diff
@@ -15,208 +15,222 @@
+            ></a>
+          </li>
+          <li>
+            <a class="social" href="https://twitter.com/AllassaneDicko0/"
+              ><i class="fa-brands fa-twitter"></i
+            ></a>
+          </li>
+        </ul>
+        <a class="btn-round" href="#"><i class="fa-solid fa-angle-down"></i></a>  
+      </div>
     </section>
 
     <section id="works">
-      <h2 class="section-title">My Recent Works</h2>
-      <hr />
-      <ul>
-        <li class="work">
-          <img src="#" alt="project image" />
-          <div>
-            <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list-tag">
-              <li class="tags">Ruby on rails</li>
-              <li class="tags">css</li>
-              <li class="tags">JavaScript</li>
-              <li class="tags">html</li>
-            </ul>
-            <button class="buttons">See Project</button>
-          </div>
-        </li>
-        <li class="work">
-          <img src="#" alt="project image" />
-          <div>
-            <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list-tag">
-              <li class="tags">Ruby on rails</li>
-              <li class="tags">css</li>
-              <li class="tags">JavaScript</li>
-              <li class="tags">html</li>
-            </ul>
-            <button class="buttons">See Project</button>
-          </div>
-        </li>
-        <li class="work">
-          <img src="#" alt="project image" />
-          <div>
-            <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list-tag">
-              <li class="tags">Ruby on rails</li>
-              <li class="tags">css</li>
-              <li class="tags">JavaScript</li>
-              <li class="tags">html</li>
-            </ul>
-            <button class="buttons">See Project</button>
-          </div>
-        </li>
-        <li class="work">
-          <img src="#" alt="project image" />
-          <div>
-            <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
+      <div class="work-wrapper">
+        <h2 class="section-title">My Recent Works</h2>
+        <hr />
+        <ul class="list-work">
+          <li class="work">
+            <img src="#" alt="project image" />
+            <div>
+              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
+              <ul class="list-tag">
+                <li class="tags">Ruby on rails</li>
+                <li class="tags">css</li>
+                <li class="tags">JavaScript</li>
+                <li class="tags">html</li>
+              </ul>
+              <button class="buttons">See Project</button>
+            </div>
+          </li>
+          <li class="work">
+            <img src="#" alt="project image" />
+            <div>
+              <h2 class="section-title">Multi-Post Stories Gain+Glory</h2>
+              <ul class="list-tag">
```

Comment:
- [ ]  I think it would be better to add the image source for all the projects or remove all the images from the work section

***Your design***
![Screenshot from 2022-11-02 19-16-23](https://user-images.githubusercontent.com/65129228/199555233-5a988dca-a0d8-42b0-8799-79eb0fab44ce.png)